doc/install.rst

```diff
@@ -4,40 +4,52 @@
 Installing scikit-learn
 =======================
 
-.. note::
+There are different ways to install scikit-learn:
+
+  * :ref:`Install the latest official release <install_official_release>`. This
+    is the best approach for most users. It will provide a stable version
+    and pre-built packages are available for most platforms.
 
-    If you wish to contribute to the project, it's recommended you
-    :ref:`install the latest development version<install_bleeding_edge>`.
+  * Install the version of scikit-learn provided by your
+    :ref:`operating system or Python distribution <install_by_distribution>`.
+    This is a quick option for those who have operating systems or Python
+    distributions that distribute scikit-learn.
+    It might not provide the latest release version.
+
+  * :ref:`Building the package from source
+    <install_bleeding_edge>`. This is best for users who want the
+    latest-and-greatest features and aren't afraid of running
+    brand-new code. This is also needed for users who wish to contribute to the
+    project.
 
 
 .. _install_official_release:
 
 Installing the latest release
 =============================
 
-Scikit-learn requires:
+If you already have a working installation of numpy and scipy,
+the easiest way to install scikit-learn is using ``pip`` or ``conda``.
 
-- Python (>= 3.5)
-- NumPy (>= 1.11.0)
-- SciPy (>= 0.17.0)
-- joblib (>= 0.11)
+With ``pip``, in order to avoid potential conflicts with other packages it is
+strongly recommended to use a virtual environment, e.g. python3 ``virtualenv``
+(see `python3 virtualenv documentation
+<https://docs.python.org/3/tutorial/venv.html>`_).
 
-Scikit-learn plotting capabilities (i.e., functions start with "plot\_"
-and classes end with "Display") require Matplotlib (>= 1.5.1). For running the
-examples Matplotlib >= 1.5.1 is required. A few examples require
-scikit-image >= 0.12.3, a few examples require pandas >= 0.18.0.
+That gives, on Windows::
 
-.. warning::
-
-    Scikit-learn 0.20 was the last version to support Python 2.7 and Python 3.4.
-    Scikit-learn now requires Python 3.5 or newer.
+    python -m venv .myenv
+    .myenv\Scripts\activate
+    pip install -U scikit-learn
 
-If you already have a working installation of numpy and scipy,
-the easiest way to install scikit-learn is using ``pip`` ::
+or on Linux::
 
+    python3 -m venv .myenv
+    source .myenv/bin/activate
     pip install -U scikit-learn
 
-or ``conda``::
+If you choose to use ``conda`` (see the `instructions for downloading conda
+<https://docs.conda.io/projects/conda/en/latest/user-guide/install/download.html>`_)
```

Comment:
Please also link to the documentation to use conda environments: https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html